routes/web.php

```diff
@@ -22,3 +22,11 @@
 Route::resource('projects', 'ProjectController')->only([
 	'index', 'show', 'create', 'store'
 ]);
+
+Route::get('/course-create', function() {
+	return view('createCourse');
+});
+
+Route::get('/success', function() {
```

Comment:
hmm do we want a whole new view for success? otra opción que se me ocurre sería encerrar el create y el success en un div y sólo cambiar hacerlos hidden por CSS (display: none) pero la verdad no sé qué sea mejor.

en cualquier caso, si vas a dejar éste route, yo haría algo como "course/create/success" 🐙 

@hecerinc: tú sabes cuál sería mejor práctica? what would Jesus do?

Reply:
Hoy en la tarde estábamos comentando con @melytc y @hecerinc que lo mejor sería regresar este view nuevo con la course key desde el back-end, please correct me if I misunderstood.

Reply:
![Jesus](https://media3.giphy.com/media/ibGFpMv1Uoais/giphy.gif?cid=3640f6095c78c23175455730770f86b7)

Sii todo lo de routes lo ve @osdagoso. Esto iría adentro del controlador en un path de success. Your suggestion sounds good.